src/index.html

```diff
@@ -3,11 +3,59 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div
+        href="#"
+        class="card__image">
+      </div>
+
+      <div class="card__item">
+        <h2 class="card__item-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <h2 class="card__item-code">
+          Product code: 195434
+        </h2>
+      </div>
+
+      <div class="card__review"
+        data-qa="card">
+
+        <div class="stars stars--4">
+          <div href="#" class="stars__star"></div>
```

Comment:
The div doesn't have the href attrubute, need to remove it
```suggestion
          <div class="stars__star"></div>
```